init.lua

```diff
@@ -1,11 +1,40 @@
 -- This file includes common functionality that should be available globally
+if Spring.CommonFunctionsInitialized then
+	return
+end
+
+local environment = Script.GetName and Script.GetName() or "LuaParser"
+
+local commonFunctions = {
+	spring = {
+		LuaMenu   = true,
+		LuaIntro  = true,
+		LuaParser = true,
```

Comment:
Let's inscribe the recently gained knowledge:
```suggestion
		LuaParser = true, -- note: LuaParser has a castrated Spring table with just GetModOptions,
		                  -- perhaps the wrapper should be a separate file too so that LuaParser
		                  -- won't get the other Spring stuff that it can't use
```

Reply:
I'll extract base Spring overrides to a separate file, so they can be executed separately.

Reply:
Spring overrides have been separated, notes have been added on what should be removed when